docs/reference/tab-widgets/troubleshooting/data/add-tier.asciidoc

```diff
@@ -0,0 +1,127 @@
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT my-index-000001
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE my-index-000001
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+
+// tag::cloud[]
+In order to get the shards assigned we need enable a new tier in the deployment.
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is 
+{cloud}/ec-access-kibana.html[enable it first].
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Dev Tools > Console**.
++
+[role="screenshot"]
+image::images/kibana-console.png[{kib} Console,align="center"]
+
+. Inspect which tier is an index targeting for assignment. <<indices-get-settings, Retrieve>>
+the configured value for the `index.routing.allocation.include._tier_preference`
+setting:
++
+[source,console]
+----
+GET /my-index-000001/_settings/index.routing.allocation.include._tier_preference?flat_settings
+----
++
+The reponse will look like this:
++
+[source,console-result]
+----
+{
+  "my-index-000001": {
+    "settings": {
+      "index.routing.allocation.include._tier_preference": "data_warm,data_hot" <1>
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
++
+<1> Represents a comma separated list of data tier node roles this index is allowed
+to be allocated on, the first one in the list being the one with the higher priority
+i.e. the tier the index is targeting.
+e.g. in this example the tier preference is `data_warm,data_hot` so the index is
+targeting the `warm` tier and more nodes with the `data_warm` role are needed in
+the {es} cluster.
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Manage this deployment**.
+
+. From the right hand side, click to expand the **Manage** dropdown button and
+select **Edit deployment** from the list of options.
+
+. On the **Edit** page, click on **+ Add Capacity** for the tier you identified
+you need to enable in your deployment. Choose the desired size and availability
+zones for the new tier.
```

Comment:
I'm not entirely sure autoscaling handles this case? I'll test it and report back

Reply:
So, if autoscaling is already enabled when we have an unassigned shard due to missing tier, autoscaling will indeed add capacity for the target tier.

However, we can't recommend enabling autoscaling when we have an unassigned shard due to missing tier because the plan change will fail (as the cluster health is RED)

Reply:
Good call, thanks for clarifying that!